client/src/components/TestSuite/TestSuiteTree/TreeItemLabel.tsx

```diff
@@ -4,18 +4,19 @@ import { Typography, Box } from '@mui/material';
 import useStyles from './styles';
 
 export interface TreeItemLabelProps {
-  runnable: TestSuite | TestGroup;
+  runnable?: TestSuite | TestGroup;
+  title?: String;
 }
 
-const TreeItemLabel: FC<TreeItemLabelProps> = ({ runnable }) => {
+const TreeItemLabel: FC<TreeItemLabelProps> = ({ runnable, title }) => {
   const styles = useStyles();
   return (
-    <Box className={styles.labelRoot} data-testid={`tiLabel-${runnable.id}`}>
+    <Box className={styles.labelRoot} data-testid={`tiLabel-${runnable?.id}`}>
```

Comment:
Linter is complaining, I think this is the correct line?
```suggestion
    <Box className={styles.labelRoot} data-testid={`tiLabel-${runnable?.id as string}`}>
```